ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/title/v1/title/title.html

```diff
@@ -17,6 +17,7 @@
      data-sly-use.template="core/wcm/components/commons/v1/templates.html"
      data-sly-test.text="${title.text}"
      id="${title.id}"
+     data-cmp-data-layer="${title.data.json}"
```

Comment:
Can we add test cases specific to data layer in cypress ?

Reply:
added cypress test cases for datalayer in the PR: https://github.com/adobe/aem-core-forms-components/pull/556

Reply:
Can you also add unitTest cases in sling model for data layer ? Schema validation should fail since you have added a new property called `dataLayer`, you also need to update af2-docs with this new property.

Reply:
It seems that the data layer schema is going via this PR: https://git.corp.adobe.com/livecycle/af2-docs/pull/173/files

Reply:
I will add datalayer wiki via: https://jira.corp.adobe.com/browse/FORMS-8170 tentatively before may release.